sim_config:
  topics:
    magnet_pose_topic: "/magnet_pose/sim_true"
    magnetic_field_topic: "/mag_sensor/data_mT"
  magnet:
    strength: 10.0
    direction: [0.0, 0.0, 1.0]
  path:
    width: 0.03
    height: 0.03
    z: 0.03
    center:
      x: 0.02
      y: 0.02
    update_rate: 100.0
    velocity: 0.01
  motion:
    mode: translate_only
    axis: z
    angular_velocity: 0.1
    initial_roll: 0.0
    initial_pitch: 0.0
    initial_yaw: 0.0
    static_position: [0.02, 0.02, 0.03]
    static_orientation: [0.0, 0.5, 0.0]
  noise:
    enable: false
    type: gaussian
    mean: 0.0
    stddev: 1.0
    amplitude: 1.0
  
  # 传感器阵列定义改为从 launch 加载的共享文件 sensor_array.yaml

